CMakeLists.txt

```diff
@@ -118,12 +116,16 @@ if (HDF5_ENABLE_THREADSAFE AND NOT Threads_FOUND)
   find_package(Threads)
 endif ()
 
-find_package (CURL 7.61 REQUIRED)
+set (CURL_TARGET_LIBTYPE shared)
+
+set(CURL_NO_CURL_CMAKE ON)
```

Comment:
This seems like something that really should only be set for Windows and not other platforms

Reply:
That said, it's mentioned that curl is often built with cmake these days anyway, so setting this to ON shouldn't necessarily be needed in the first place